e2e/tests/adminManagesOrders_test.js

```diff
@@ -7,7 +7,8 @@ const caseDataWithApplication = require('../fixtures/caseData/gatekeepingWithPas
 const closedCaseData = require('../fixtures/caseData/closedCase.json');
 
 const orderTitle = 'some title';
-const today = moment().hours(10).minutes(5).seconds(23).milliseconds(0).toDate();
+const now = moment().toDate();
+const today = moment(now).subtract(1, 'hours').minutes(5).seconds(23).milliseconds(0).toDate();
```

Comment:
if must be in past, then maybe safer to subtract 2 hours, otherwise with summer time it can produce incorrect result, e.g test run on local machine with summer time 14:03 , server in UTC with 13:03